src/elements/element.html

```diff
@@ -49,10 +60,10 @@
       // (2) element default values
       static get _flattenedProperties() {
         if (!this.hasOwnProperty('__flattenedProperties')) {
-          this.__flattenedProperties = this._ownConfig.properties || {};
+          this.__flattenedProperties = flattenProperties({}, this._ownConfig.properties);
```

Comment:
Let's add a comment here that we need to scrutinize this when optimizing. There's an extra loop that we didn't need before and the custom `flattenProperties` prevents us from potentially using `Object.assign`. It's probably nbd but I feel safer putting a bookmark in the code that we made a decision here.